requirements/base.txt

```diff
@@ -244,10 +248,14 @@ pyyaml==5.4.1
     #   apispec
 redis==3.5.3
     # via apache-superset
+requests==2.27.1
+    # via shillelagh
 sasl==0.3.1
     # via -r requirements/base.in
 selenium==3.141.0
     # via apache-superset
+shillelagh==1.0.13
```

Comment:
isn't it an optional package?

Reply:
It's needed for the native dialect. Let me add it as an optional dep.